Classes/Service/FileGenerator.php

```diff
@@ -240,7 +238,11 @@ public function build(Extension $extension)
             $this->generateDocumentationFiles();
         }
 
-        $this->generateEmptyGitRepository();
+        $extensionSettings = $extension->getSettings();
+        if ($extensionSettings['overwriteSettings']['Configuration']['GIT']) {
```

Comment:
since there is no default value, I assume this is a change in behaviour as no git-files are generated.
besides that the condition needs to be `if ($extensionSettings['overwriteSettings']['Configuration']['GIT'] ?? false)` to avoid PHP notices

Reply:
Yes indeed it is to change the behaviour of not having git-files generation. Good point for the PHP notice, let's change it.

Reply:
IMO *by default* git files should be generated. Dev`s should be encouraged to use git.
@nicodh your opinion as well?